GooglePlayInstant/Samples/TestApp/Editor/TestAppBuilder.cs

```diff
@@ -20,14 +20,13 @@
 namespace GooglePlayInstant.Samples.TestApp.Editor
 {
     /// <summary>
-    /// Exposes build and run functionality to the command line for testing.
+    /// Exposes build functionality to the command line for testing.
     /// </summary>
     public static class TestAppBuilder
     {
         private const string BundleIdentifier = "com.google.android.instantapps.samples.unity.testapp";
-        private const string ApkPath = "Assets/TestApp.apk";
-        private const string ScenesPath = "Assets/TestApp/Scenes/";
-        private static readonly string[] SceneFilesToTest = {"TestScene.unity"};
+        private const string ApkPath = "Assets/../TestApp.apk";
```

Comment:
Is it possible to pass this is in as a parameter from the command line?

Reply:
It's definitely possible. Do you want me to include that in this PR, or split it up into a separate one?

Reply:
Maybe separate just so you can submit this PR as-is?

Reply:
sounds good